.github/pull_request_template.md

```diff
@@ -16,13 +16,16 @@
     of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
     description heading and to add a 'x' to the correct box.
 -->
-    [ ] Fix
-    [ ] Refactor
-    [ ] Target update
-    [ ] Functionality change
-    [ ] Docs update
-    [ ] Test update
-    [ ] Breaking change
+    [ ] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
+    [ ] Feature update (New feature / Functionality change / New API)
+    [ ] Major update (Breaking change E.g. Return code change / API behaviour change)
```

Comment:
should we add here `- Release notes required` , and also for feature update?

Reply:
There is a comment under the Release Notes section that says it is compulsory for Feature and Major updates

Reply:
> There is a comment under the Release Notes section that says it is compulsory for Feature and Major updates

yes, just thinking if I fill this in if I get to Release notes :-) I would leave it in both places